packages/shared-data/config.ts

```diff
@@ -0,0 +1,104 @@
+const config = {
+  auth: {
```

Comment:
is this all auth config? should it be called that?

Reply:
auth defaults is probably more accurate, lemme change 👍🏼